Week2/exercises/arrayExercise.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+var array = [1, 2, 3, 4, 5];
+
+//Write a JavaScript function to get the first element of an array.
+var getFirstElement = (arr) => {
+  return arr.shift();
+}
+
+//...Passing a parameter 'n' will return the first 'n' elements of the array.
+var getFirstNElements =  (array, n) => {
```

Comment:
try to use a default parameter here
getFirstElement & getFirstNElements should be a single function

Reply:
default parameter added and now in one function.